linter/src/LintCommand.php

```diff
@@ -65,6 +65,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
             return $this->error ? 1 : 0;
         }
 
+        $this->validatePackageNames();
```

Comment:
This should be moved above the if-condition. We want to validate the folder structure for pull requests as well.